daprdocs/content/en/reference/resource-specs/resiliency-schema.md

```diff
@@ -0,0 +1,60 @@
+---
+type: docs
+title: "Resiliency spec"
+linkTitle: "Resiliency spec"
+weight: 3000
+description: "The basic spec for a Dapr resiliency resource"
+---
+
+The `Resiliency` Dapr resource allows you to define and apply fault tolerance resiliency polcies. Resiliency specs are applied with the Dapr sidecar starts. 
+
+```yml
+apiVersion: dapr.io/v1alpha1
+kind: Resiliency
+metadata:
+  name: <REPLACE-WITH-RESOURCE-NAME>
+scopes:
+  - <REPLACE-WITH-SCOPED-APPIDS>
+spec:
+  policies: # Required
+    timeouts: # Replce with any unique name
+      timeoutName: <REPLACE-WITH-TIME-VALUE>
+    retries:
+      retryName: # Replace with any unique name
+        policy: exponential
+        maxInterval: <REPLACE-WITH-VALUE>
+        maxRetries: <REPLACE-WITH-VALUE>
+    circuitBreakers:
+      circuitBreakerName: # Replace with any unique name
+        maxRequests: <REPLACE-WITH-VALUE>
+        timeout: <REPLACE-WITH-VALUE> 
+        trip: <REPLACE-WITH-CONSECUTIVE-FAILURE-VALUE>
+targets: # Required
+    apps:
+      appID: # Replace with scoped app ID
+        timeout: <REPLACE-WITH-TIMEOUT-NAME>
+        retry: <REPLACE-WITH-RETRY-NAME>
+        circuitBreaker: <REPLACE-WITH-CIRCUIT-BREAKER-NAME>
+    actors:
+      myActorType: 
+        timeout: <REPLACE-WITH-TIMEOUT-NAME>
+        retry: <REPLACE-WITH-RETRY-NAME>
+        circuitBreaker: <REPLACE-WITH-CIRCUIT-BREAKER-NAME>
+        circuitBreakerCacheSize: <REPLACE-WITH-VALUE>
+    components:
+      componentName: # Replace with your component name
+        outbound:
+          timeout: <REPLACE-WITH-TIMEOUT-NAME>
+          retry: <REPLACE-WITH-RETRY-NAME>
+          circuitBreaker: <REPLACE-WITH-CIRCUIT-BREAKER-NAME>
+```
+
+## Spec fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
+| policies | Y | The configuration of resiliency policies, including: <br><ul><li>`timeouts`</li><li>`retries`</li><li>`circuitBreakers`</li></ul> | timeout: `general`<br>retry: `retryForever`<br>circuit breaker: `simpleCB` |
+| targets | Y | The configuration for the applications, actors, or components that use the resiliency policies. |  |
```

Comment:
I would add a link to here https://docs.dapr.io/operations/resiliency/targets/ directly

Reply:
It would also be good to see you can have a version that includes all the built-in retry Policies (these 4), so that these are seen in one place. These https://docs.dapr.io/operations/resiliency/policies/

Reply:
And also worth pointing to the other default policies. 
In fact what do you think about breaking the default policies section into its own topic? Good idea or not?

Reply:
@msfussell like breaking it out from the policies doc and having it sit under operations/resiliency? that could work. I'd pull out both 'setting default policies' and 'overriding default policies' sections and put in their own doc.